client/packages/invoices/src/Prescriptions/api/hooks/useInsurance.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@openmsupply-client/common';
+import { usePrescriptionGraphQL } from '../usePrescriptionGraphQL';
+import { INSURANCE, LIST } from './keys';
+
+export const useInsurance = (
+  insuranceId: string | undefined | null,
+  storeId: string
+) => {
+  const { prescriptionApi } = usePrescriptionGraphQL();
+
+  const queryKey = [INSURANCE, LIST, { insuranceId, storeId }];
+
+  const queryFn = async () => {
+    const query = await prescriptionApi.insuranceById({
+      storeId,
+      insuranceId: insuranceId ?? '',
```

Comment:
This will return an error if insuranceId is an empty string. We should throw and error to help handle this case

Reply:
They query won't run on empty string as per below. I think it's ok, although is needing a refactor...